library/network_connections.py

```diff
@@ -1299,7 +1299,14 @@ def check_deactivated_cb():
             finally:
                 ac.handler_disconnect(ac_id)
 
+    def reapply(self, interface, connection):
+        device = self.nmclient.get_device_by_iface(interface)
+        if not device:
+            return False
 
+        version_id = 0
+        flags = 0
+        return Util.call_async(device, "reapply", [connection, version_id, flags])
```

Comment:
`Util.call_async()` is a bit of a misnomer, because the makes an async API call and waits for it to complete. It is syncronously, but the `_async` prefix sounds like it would be async. But I don't have a better suggestions, so ok.

Reply:
Interesting, I used the `async` suffix because I want to call something that is async... Maybe call it `call_async_method()` instead?

Reply:
`call_async_synchronously()` ?

Reply:
I am fine with all suggestions. Just wanted to point it out. Please choose what you find best.

The "_async()" and "_sync()" suffixes you often find in glib and libnm, to indicate the nature of the function.

Reply:
Changed it to `call_async_method()`.